.NET/Microsoft.Recognizers.Text.NumberWithUnit/Parsers/BaseCurrencyParser.cs

```diff
@@ -175,9 +175,16 @@ private ParseResult MergeCompoundUnit(ExtractResult compoundResult)
                     // Match pure number as fraction unit.
                     if (extractResult.Type.Equals(Constants.SYS_NUM, StringComparison.Ordinal))
                     {
-                        numberValue += (double)parseResult.Value * (1.0 / 100);
-                        result.ResolutionStr += ' ' + parseResult.ResolutionStr;
-                        result.Length = parseResult.Start + parseResult.Length - result.Start;
+                        Config.SpecialCurrencyFractionMaxMap.TryGetValue(mainUnitIsoCode, out var fractionMaxValue);
+
+                        fractionMaxValue = fractionMaxValue == 0 ? 100 : fractionMaxValue;
```

Comment:
Can you declare a constant in this class for this usage of 100? Something like DEFAULT_FRACTIONAL_SUBUNIT?